modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,42 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parsePelSummary(stream: DataStream, config: Config):
+    out = OrderedDict()
+    ret, ph = generatePH(stream, out)
+    if ret is False:
+        return "", ""
+    eid = ph.pLID
+    ret, uh = generateUH(stream, ph.creatorID, out)
+    if ret is False:
+        return "", ""
+    
+    if config.serviceable_only and not uh.isServiceable():
+        return "", ""
+
+    if config.non_serviceable_only and uh.isServiceable():
```

Comment:
Same as LN 258

Reply:
Yeah I understood, requesting to add some details in the function documentation what it will return. Its your wish too if you think code explain that.